libass/ass.c

```diff
@@ -81,6 +81,23 @@ struct parser_priv {
     uint32_t sinfo_flags;
 };
 
+static const char *ASS_STYLE_FORMAT =
```

Comment:
Not sure about all uppercase here. Personally I think that it should be all lowercase instead, but I'm not an expert on libass style conventions.

Reply:
libass does indeed seem to use lowercase for `static const`. Thanks!
Done.

Reply:
(_Technically,_ this isn’t even `static const`: it’s a nonconst pointer to const character.)

Reply:
They probably should be const, thanks.
Pushed
```diff
-static const char *ass_style_format =
+static const char *const ass_style_format =
```
for all format line defaults